src/sentry/demo/data_population.py

```diff
@@ -100,6 +103,23 @@
     },
 }
 
+saved_search_by_platform = {
+    "global": [
+        ["Unhandled Errors", "is:unresolved error.unhandled:true"],
+        ["Release 3.2 Errors", "release:checkout-app@3.2"],
+    ],
+    "python": [
+        ["Firefox Errors - Python", "browser.name:Firefox"],
+    ],
+    # "javascript-react": [["Edge Errors - React", "browser.name:Edge"]],
+    "apple-ios": [["iOS 12 Errors", 'os:"iOS 12"']],
+    "javascript-react": [],
+    "android": [],
+    "react-native": []
+    # "android": [["Pixel Device Errors", "device.family:Pixel"]],
+    # "react-native": [["Handled Errors - React Native", "error.unhandled:false"]],
```

Comment:
@roggenkemper whats up with the commented-out lines?

Reply:
I had them but I thought it looked too cluttered. do you think it's worth including? I commented out since I wasn't sure if we wanted them.

Reply:
@roggenkemper up to you, but we should remove the comment if we don't want it